mitodl/open-discussions
# Remove algolia places, LocationPicker component, and profile location field

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #3498 

#### What's this PR do?
Removes code for letting the user choose a profile location via the Algolia Places API.

#### How should this be manually tested?
Create or login as a user.  Go to the profile page.  There should not be a location displayed.  Edit your profile, there still should not be a location field.  Make changes and submit, the changes should be saved without errors.


#### Any background context you want to provide?
The Algolia Places API will be deactivated at the end of the month.  Eventually we should pick a new API to replace it, but after discussion with Ferdi it was decided that we should just remove the location field and associated API integration code for now.
